themes/custom/lex/patterns/molecules/icon-button-with-description.twig

```diff
@@ -0,0 +1,29 @@
+<a href="{{ url }}">
+    <div class="icon-desc-button">
+        <div class="row">
+            {% if icon[0]['#text']!= null %}
+                <div class="col-2">
+                    {% include '@components/atoms/button-icon.twig' with {icon: icon} %}
+                </div>
+                <div class="col-10">
+                    {% include '@components/atoms/button-heading.twig' with {heading: heading} %}
+                    {% if description[0]['#text']!= null %}
+                        {% include '@components/atoms/description.twig' with {description: description} %}
+                    {% endif %}
+                </div>
+            {% else %}
+                <div class="col-10">
+                    <span class="text-center">{% include '@components/atoms/button-heading.twig' with {heading: heading} %}</span>
+                    {% if description[0]['#text']!= null %}
+                        {% include '@components/atoms/description.twig' with {description: description} %}
+                    </div>
+                    {% else %}
+                    </div>
```

Comment:
Tried to cut down a little on duplicated code. See if this works for this file. It may not but there was just some of the same checks going on so I thought we could simplify it and reduce the repetition.

```suggestion
<a href="{{ url }}">
    <div class="icon-desc-button">
        <div class="row">
            {% if icon[0]['#text']!= null %}
                <div class="col-2">
                    {% include '@components/atoms/button-icon.twig' with {icon: icon} %}
                </div>
            {% endif %}
            <div class="col-10">
              <span class="text-center">
                {% include '@components/atoms/button-heading.twig' with {heading: heading} %}
              </span>
              {% if description[0]['#text']!= null %}
                {% include '@components/atoms/description.twig' with {description: description} %}
              {% else %}
                <div class="col-1">
                    <span class="button-arrow"><i class="fas fa-chevron-right"></i></span>
                </div>
              {% endif %}
            </div>
        </div>
    <div>
</a>
```